SFS_general/easyfft.m

```diff
@@ -101,5 +101,11 @@
 
 %% ===== Plotting ========================================================
 if nargout==0 || useplot
-    figure; semilogx(f,20*log10(abs(amplitude)));
+    figure; title('Spectrum');
+    subplot(2,1,1) 
+    semilogx(f,20*log10(abs(amplitude))); xlim([1 fs/2]);
+    grid on; xlabel('Frequency [Hz]'); ylabel('Amplitude [dB]')
```

Comment:
[Frequncy] = Hz
[Hz] = ???
So, please change this to Frequency / Hz and Amplitude / dB.